CMakeLists.txt

```diff
@@ -5,3 +5,5 @@ find_package(Rock)
 rock_activate_cxx11()
 rock_init(pocolog_cpp 0.1)
 rock_standard_layout()
+
+add_compile_options(-DBASE_LOG_NAMESPACE=${PROJECT_NAME})
```

Comment:
Already part of [rock_init](https://github.com/rock-core/base-cmake/blob/7c60186292d3a18794ef6f7fb7d23e1f72ed5d56/modules/Rock.cmake#L130)

Reply:
Fixed